cluster/gce/config-default.sh

```diff
@@ -34,17 +34,16 @@ REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-true}
 PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 PREEMPTIBLE_MASTER=${PREEMPTIBLE_MASTER:-false}
 
-# TODO(#26183): Provide a way to differentiate master OS distro and node OS
-# distro.
-OS_DISTRIBUTION=${KUBE_OS_DISTRIBUTION:-gci}
+MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-gci}
```

Comment:
This breaks backwards compatibility with anyone who currently uses `KUBE_OS_DISTRIBUTION`.

Shouldn't this instead be something like `MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-${KUBE_OS_DISTRIBUTION:-gci}}` to preserve backwards compatibility?

It seems quite sane to me to default to KUBE_OS_DISTRIBUTION for both master and node, with override provided by their specific environment variables.